### ISSUE TITLE
Progress bar appears twice

### ISSUE THREAD

**Issue by ceremcem** [loadingio/loading-bar#6]
Livescript [code](https://github.com/aktos-io/scada.js/blob/72de1d8fcf724e349fee1b0978ced87c019e6b91/src/client/components/progress/index.ls#L23-L24) for initialization ldBar: 

```ls
        bar = new ldBar '#myselector', do
            "stroke": '#f00'
```

HTML [part](https://github.com/aktos-io/scada.js/blob/72de1d8fcf724e349fee1b0978ced87c019e6b91/src/client/components/progress/index.pug#L1) (in pug): 

```pug
dive(id="myselector")
```

Result: 

![image](https://user-images.githubusercontent.com/6639874/27534306-3f2d8434-5a6f-11e7-9f50-0817770b3529.png)

**Comment by ceremcem** [comment]
[These lines](https://github.com/loadingio/loading-bar/pull/9/commits/5ed3d3866540febc0a6dc75d682005f31174cc4d#diff-165c185c429836b045dd3d914aeb647eR355) resolves the issue, but also breaks "no Javascript initialization" "feature".

**Comment by dennisaskeland** [comment]
I'm having this issue as well. Any plans to release a patche to fix this?

**Comment by ceremcem** [comment]
Yes, of course. I'll probably cover this one in [this PR](https://github.com/loadingio/loading-bar/pull/9) but it's a decision up to @zbryikt to drop "no javascript initialization" feature. 

Anyway, did you try [disabling that feature](https://github.com/loadingio/loading-bar/issues/6#issuecomment-311109115), which resolves the problem for now?

**Comment by ceremcem** [comment]
You may also want to try out [this precompiled](https://raw.githubusercontent.com/aktos-io/scada.js/v0.5/vendor/loading-io/loading-bar.js) one and see if it resolves your case or not.

**Comment by dennisaskeland** [comment]
@ceremcem that precompiled one worked like a charm, thanks!

**Comment by ceremcem** [comment]
You're welcome :+1:

**Comment by zbryikt** [comment]
hi guys, sorry for taking so long to reply here.

Initialization by JS will add a 'ldBar' class to target element, while no JS init runs after document is ready, so if we run JS initialization before document is ready then there will be another bar generated by no JS init code.

but no Javascript initialization is probably still needed, so I added a check to see if there is already an ldBar instance created. related commit here: 6c83d77f3
